tests/cbmc/sources/cbmc_utils.c

```diff
@@ -194,3 +195,76 @@ void nondet_s2n_mem_init()
 {
     if (nondet_bool()) { s2n_mem_init(); }
 }
+
+void assert_rc_decrement_on_evp_pkey_ctx(struct rc_keys_from_evp_pkey_ctx *storage)
+{
+    if (storage->pkey_refs > 1)
+        assert(storage->pkey->references == storage->pkey_refs - 1);
+    else if (storage->pkey_refs == 1 && storage->pkey_eckey_refs > 1)
```

Comment:
Nit: in S2N, we always use { }, even for one-line ifs. It's clearer, and you don't have to add the { } later if this becomes two lines.